### ISSUE TITLE
Intergrate to Flipper

### ISSUE THREAD

**Issue by moonjava2005** [reduxjs/redux-devtools#506]
Hi you guys,
There's any way to integrate redux-devtools to Flipper.
I tried to search on stackoverflow but there's no clearly solution
Thanks for you helps

**Comment by timdorr** [comment]
You probably want https://github.com/jk-gan/redux-flipper. This project is geared towards Dev Tools implementations.

**Comment by savelichalex** [comment]
@timdorr not exactly. For example it won't work with [react-navigation](https://reactnavigation.org) because of [this](https://github.com/react-navigation/react-navigation/blob/main/packages/devtools/src/useReduxDevToolsExtension.tsx#L33), as `react-navigation` doesn't have explicit store to apply middleware (actually no redux store at all, but they send compatible events to devtools).

**Comment by christophemenager** [comment]
@savelichalex is correct, react-navigation does not work with the current redux-flipper. 
However, this project could use some help to fix this : https://github.com/jk-gan/redux-flipper/issues/38
